SendLog.py

```diff
@@ -1668,6 +1669,43 @@ def run_scilab_func_request():
     return list_value
 
 
+# App route for getting scilab expression output for Expression Block
+
+@app.route('/getExpressionOutput', methods=['POST'])
+def run_scilab_func_expr_request():
+    (diagram, __) = get_diagram(get_request_id())
+    if diagram is None:
+        print('no diagram')
+        return
+
+    pathfortxtfile = diagram.sessiondir;
+    head = request.form['head']
+    exx = request.form['exx']
+    command = "exec('" + EXP_SCI_FUNC_WRITE + \
+    "');callFunctionAcctoMethod('" + pathfortxtfile + \
```

Comment:
```suggestion
        "');callFunctionAcctoMethod('" + pathfortxtfile + \
```

as this is a continuation line, it should be indented by one tab